packages/terra-dropdown-button/src/Item.jsx

```diff
@@ -11,7 +11,13 @@ const propTypes = {
    */
   label: PropTypes.string.isRequired,
   /**
-   * The function that is called when the option is clicked or enter/space is pressed with it focused.
+   * The associated metaData to be provided in the onSelect callback.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  metaData: PropTypes.object,
+  /**
+   * Function callback for when the appropriate click or key action is performed.
```

Comment:
I'd keep the old description as-is.

Reply:
Keeping it consistent from [here](https://github.com/cerner/terra-core/blob/e57d0b5e93e51d35c689b65fbe5dbe76ab14e91c/packages/terra-list/src/ListItem.jsx#L33).